manifests/webhook.yaml

```diff
@@ -6,12 +6,21 @@ metadata:
   labels:
     app: admission-webhook
 rules:
-- apiGroups: ["apps"]
-  resources: ["statefulsets"]
-  verbs: ["get"]
-- apiGroups: ["pingcap.com"]
-  resources: ["tidbclusters"]
-  verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["create", "get", "list", "watch", "update"]
```

Comment:
permissions on endpoints can be removed because we don't use leader election now

Reply:
fix